packages/plugin-suno/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "@elizaos/plugin-suno",
+  "version": "0.1.9-alpha.1",
+  "description": "Suno AI Music Generation Plugin for Eliza",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@elizaos/core": "^0.1.0"
+  },
+  "devDependencies": {
+    "typescript": "^4.9.0",
+    "@types/node": "^16.0.0",
+    "jest": "^27.0.0",
+    "@types/jest": "^27.0.0"
+  },
+  "peerDependencies": {
+    "@elizaos/eliza": "^0.1.0"
+  }

```

Hi! I believe you're looking for the @elizaos/core package here which is already listed so this can be removed I believe
```suggestion
  }
```